pkg/cmd/uninstall.go

```diff
@@ -105,7 +107,7 @@ func (o *uninstallCmdOptions) uninstall(cmd *cobra.Command, _ []string) error {
 	}
 
 	uninstallViaOLM := false
-	if o.OlmEnabled {
+	if o.OlmEnabled || o.UninstallAll {
```

Comment:
```suggestion
	if o.OlmEnabled {
```

We should not do olm uninstall e.g. if OLM is not installed in the cluster

Reply:
Good point